### PR TITLE
Fixed bug with MITRE ATT&CK dashboard

### PR DESCRIPTION
**Pull request by mayagoldb**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/26834

## Description
- Replaced the use of Demisto REST API with demisto.getIndicators (removed this from dependencies)
- Handled issue when empty phase_name caused the widget to keep loading without results.

## Minimum version of Demisto
- [x] 6.0.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

### CODE REVIEW

**Comment by mayagoldb** [comment]
@yaron-libman please review release notes

File: Packs/FeedMitreAttack/pack_metadata.json [hunk 1]
```diff
@@ -16,5 +16,11 @@
         "MITRE",
         "Feed",
         "ATT&CK"
-    ]
+    ],
+    "dependencies": {
```
Comment (yaakovi) [inline Packs/FeedMitreAttack/pack_metadata.json:20]: why do you need to add this?

Comment (mayagoldb) [reply Packs/FeedMitreAttack/pack_metadata.json:20]: I ran the dependencies command to make sure Demisto REST API is not mandatory anymore, should this be removed?

Comment (yaakovi) [reply Packs/FeedMitreAttack/pack_metadata.json:20]: yes

**Pull request merged**